src/ripple/protocol/STObject.h

```diff
@@ -401,7 +401,7 @@ class STObject
         if (!cf)
             throw std::runtime_error ("Wrong field type");
 
-        cf->setValue (value);
+        cf->setValue (std::move (value));
```

Comment:
@seelabs Thanks for accommodating.  The interface to `setFieldUsingSetValue` is such that `T` is explicitly set by the caller.  However `V` is inferred by the passed in parameter.  So `V` could be fouled up more easily.  Yeah, Howard is (of course) right, asserting on `is_lvalue_reference` will do the job nicely.